src/Dispatcher.php

```diff
@@ -65,7 +65,7 @@ public function __invoke(RequestInterface $request)
                 $function = (new ReflectionClass($controller))->getShortName();
 
                 $eventName = sprintf('%s.%s', strtolower($function), $method);
```

Comment:
ahah php mystery..
I know that this proposal broke compatibility but I think to change

`function.method` with `class.method`.. In this way we can replace 

```php
$eventManager->attach("indexcontroller.index", ...)
```
with
```php
$eventManager->attach(IndexController::class.".index", ...)
```
Maybe this approach is more readable/writable.. What do you think?